nativelib/src/main/scala/scala/scalanative/native/errno.scala

```diff
@@ -13,4 +13,6 @@ object errno {
   def EILSEQ: CInt = extern
   @name("scalanative_erange")
   def ERANGE: CInt = extern
+  @name("scalanative_eexist")
+  def EEXIST: CInt = extern
```

Comment:
Shouldn't `errno` be moved to `scalanative.posix` altogether then? http://pubs.opengroup.org/onlinepubs/9699919799/